File: tests/fixtures/9495/test/Parse.java
public class Parse implements ParseAndCallSolve {
    @Override
    public String parseAndCallSolve(Solution sol, String input) {
        String[] lines = input.trim().split("\n");
        int n = Integer.parseInt(lines[0].trim());
        String[] board = new String[n];
        for (int i = 0; i < n; i++) board[i] = lines[i + 1].trim();
        return String.valueOf(sol.solve(n, board));
    }
}

File: tests/fixtures/9495/test/parse.py
class Parse:
    def parse_and_solve(self, sol, input: str) -> str:
        lines = input.strip().split('\n')
        n = int(lines[0].strip())
        board = [lines[i + 1].strip() for i in range(n)]
        return str(sol.solve(n, board))
